server/fleet/errors.go

```diff
@@ -540,8 +540,8 @@ const (
 	RunScriptHostOfflineErrMsg             = "Script can’t run on offline host."
 	RunScriptHostNotFoundErrMsg            = "Host doesn’t exist. Make sure you provide a valid hostname, UUID, osquery host ID, or node key."
 	RunScriptForbiddenErrMsg               = "You don’t have the right permissions in Fleet to run the script."
-	RunScriptAlreadyRunningErrMsg          = "A script is already running on this host. Please wait about 1 minute to let it finish."
-	RunScriptHostTimeoutErrMsg             = "Fleet hasn’t heard from the host in over 1 minute. Fleet doesn’t know if the script ran because the host went offline."
+	RunScriptAlreadyRunningErrMsg          = "A script is already running on this host. Please wait about 5 minutes to let it finish."
+	RunScriptHostTimeoutErrMsg             = "Fleet hasn’t heard from the host in over 5 minutes. Fleet doesn’t know if the script ran because the host went offline."
```

Comment:
That's weird, I thought those had already been changed? In any case, I adjusted the messages for the new timeout.

Reply:
They _were_ changed indeed, could it be there were some git conflict resolution issues? https://github.com/fleetdm/fleet/pull/15779/files#diff-db1c84c0a64348ea4d83b4b8b22a455b828d8fab3afb30e7aa55a01cbfcd2313R544

Reply:
that's a bit unsettling 😬